forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -1603,6 +1603,12 @@ public final CardCollectionView mill(int n, final ZoneType destination,
             }
         }
 
+        // MilledAll trigger
+        final Map<AbilityKey, Object> runParams = AbilityKey.newMap();
+        runParams.put(AbilityKey.Cards, milled);
+        runParams.put(AbilityKey.Player, this);
+        game.getTriggerHandler().runTrigger(TriggerType.MilledAll, AbilityKey.newMap(runParams), false);
```

Comment:
just runParams here

Reply:
ah makes sense – I fixed that in the source I got these lines from, too